e2e/commands/add-many.e2e.js

```diff
@@ -118,4 +123,23 @@ describe('bit add many programmatically', function () {
       expect(nodeStartOutputObj).to.be.ofSize(5);
     });
   });
+  describe('make sure gitignore is read from the correct path', function () {
+    before(function () {
+      helper.reInitLocalScope();
+      helper.copyFixtureComponents('add-many');
+      helper.linkNpm('bit-bin');
+      const newDirPath = helper.createNewDirectory();
+      const scriptRelativePath = 'add-many';
+      helper.copyFixtureComponents(scriptRelativePath, newDirPath);
+      helper.linkNpm('bit-bin', newDirPath);
+      helper.writeGitIgnore(['add_many_test_files/c.js']);
+      const scriptAbsolutePath = path.join(newDirPath, 'add_many_test_files/add_components_programmatically.js');
+      const nodeStartOutput = helper.nodeStart(`${scriptAbsolutePath} ${helper.localScopePath}`, process.cwd());
+      nodeStartOutputObj = JSON.parse(nodeStartOutput);
+      status = helper.status();
+    });
+    it('should not add a component if it is in gitignore', function () {
+      expect(status).to.not.have.string('add_many_test_files/c ... ok');
```

Comment:
do not check this string.. it's not safe. tomorrow we will change the ... ok to ...ok (without space) and your test will break.
Use status with json and check the data not the strings